packages/stepper/src/index.ts

```diff
@@ -1,6 +1,6 @@
-import Step from './Step';
-import Stepper from './Stepper';
-import type { StepperProps } from './types';
+import Step from './Step/Step';
+import Stepper from './Stepper/Stepper';
+import type { StepperProps } from './Stepper/Stepper.types';
```

Comment:
Some of your sub components have index files and some don't. To keep it consistent you could add index files to `Step` and `Stepper`, add your exports there, and then import them from `./Stepper` and `./Step`